AutomatedLabWorker/AutomatedLabVMWareWorkerNetwork.psm1

```diff
@@ -8,7 +8,7 @@ function Get-LWVMWareNetworkSwitch
 	
     Write-LogFunctionEntry
 
-    foreach ($network in $VirtualNetwork)
+    foreach ($name in $VirtualNetwork)
```

Comment:
I see. Then it should read like this (adapted from the initial original):
`foreach ($network in $VirtualNetwork)
    {
        $network = Get-VDPortgroup -Name $network.Name   # $network.Name instead of $Name
(..)
}`